ambry-protocol/src/main/java/com.github.ambry.protocol/AmbryRequests.java

```diff
@@ -167,15 +125,14 @@ public void handleRequests(Request request) throws InterruptedException {
         case DeleteRequest:
           handleDeleteRequest(request);
           break;
+        case TtlUpdateRequest:
+          handleTtlUpdateRequest(request);
+          break;
         case ReplicaMetadataRequest:
           handleReplicaMetadataRequest(request);
           break;
         case AdminRequest:
-          handleAdminRequest(request);
-          break;
-        case TtlUpdateRequest:
-          handleTtlUpdateRequest(request);
-          break;
+          throw new UnsupportedOperationException("Request type not supported");
```

Comment:
Why do we need a separate case for AdminRequest here. We can just let it goto default.